### ISSUE TITLE
exportContacts annotation

### ISSUE THREAD

**Issue by Sect0R** [unisender-dev/php-api-wrapper#10]
When translating to the English you forgot to add 

`@method exportContacts(array $params) It is a method of bulk export of contacts.`

**Comment by unisender-dev** [comment]
It may not be obvious, but there is a method **taskExportContacts**.

**Comment by Sect0R** [comment]
Yes, but old code use exportContacts methods (still present in API documentation).
Please, add it to the methods list, because IDE highlighted it as an error.
